packages/manager/apps/pci-object-storage/src/pages/object-storage/users/create/AddUserForm.component.tsx

```diff
@@ -75,15 +75,19 @@ const AddUserForm = ({ onClose }: AddUserFormProps) => {
     },
   });
 
-  const onSubmit = form.handleSubmit((formValues) => {
-    addUser({
-      projectId,
-      newUser: {
-        description: formValues.userName,
-        role: user.RoleEnum.objectstore_operator,
-      },
-    });
-  });
+  const onSubmit = (e?: React.FormEvent) => {
+    e?.preventDefault();
+    e?.stopPropagation();
+    form.handleSubmit((formValues) => {
+      addUser({
+        projectId,
+        newUser: {
+          description: formValues.userName,
+          role: user.RoleEnum.objectstore_operator,
+        },
+      });
+    })();
+  };
```

Comment:
handleSubmit actually exposes the event: https://react-hook-form.com/docs/useform/handlesubmit

So I think it would be cleaner to stop the propagation like this:

```suggestion
  const onSubmit = form.handleSubmit((formValues, e) => {
  e?.preventDefault();
  e?.stopPropagation();

  addUser({
    projectId,
    newUser: {
      description: formValues.userName,
      role: user.RoleEnum.objectstore_operator,
    },
  });
});
```